src/System.Numerics.Vectors/tests/System.Numerics.Vectors.Tests.csproj

```diff
@@ -24,7 +24,7 @@
     <Compile Include="Matrix4x4Tests.cs" />
     <Compile Include="PlaneTests.cs" />
     <Compile Include="QuaternionTests.cs" />
-    <Compile Condition="'$(TargetGroup)' == 'netfx'" Include="..\..\Common\src\System\MathF.netstandard.cs">
+    <Compile Condition="'$(TargetGroup)' != 'netcoreapp' AND '$(TargetGroup)' != 'uap'" Include="..\..\Common\src\System\MathF.netstandard.cs">
```

Comment:
Nit: It is better to use positive conditions `... == 'netfx' OR ... == 'netstandard'`

Reply:
https://github.com/dotnet/corefx/issues/16121

Reply:
This is supporting file. The reasoning in #16121 does not apply to it.